### PR TITLE
hotfix: fix some typo in graphql documentation

### PR DESCRIPTION
**Pull request by Simperfit**

### CODE REVIEW

File: core/graphql.md [hunk 1]
```diff
@@ -140,10 +140,10 @@ Or order your results like:
 
 To add a security layer to your queries and mutations, follow the [security](security.md) documentation.
 
-If your security needs differ between REST and GraphQL, add the particular parts in the `graphql` key.
+If your security needs to differ between REST and GraphQL, add the particular parts in the `graphql` key.
 
-In the example below, we want the same security rules as in REST, but we also want to allow an admin to delete a book in GraphQL only.
-Please note it's not possible to update a book in GraphQL because the `update` operation is not defined.
+In the example below, we want the same security rules as we have in REST, but we also want to allow an admin to delete a book only in GraphQL.
+Please note thats it's not possible to update a book in GraphQL because the `update` operation is not defined.
```
[the remark this answers is not in the document]
Comment (Simperfit) [reply core/graphql.md]: It's not required, but its better when reading

[the remark this answers is not in the document]
Comment (soyuka) [reply core/graphql.md]: then it should be "Please note, that it's"

[the remark this answers is not in the document]
Comment (Simperfit) [reply core/graphql.md]: yup this is outdated, look at the diff ^^

**Comment by dunglas** [comment]
Thanks @Simperfit

**Pull request merged**